lib/scope.js

```diff
@@ -435,22 +435,7 @@ module.exports = {
 
   normalizeTable: async function ( scope, { var_data }) {
     /* Return data derived from cucucmber table object or array.
-    * Turn arrays into objects and convert og story tables into
-    * our current format ( var, choice, value -> var, value, checked ) */
-
-    // let var_data = raw_var_data.hashes();
-    // let var_data = raw_var_data.hashes();
-
-    // If table doesn't have headers, give it headers
-    // No repos we know of has used the array version of this table as of 03/27/21,
-    // so we shouldn't need to support the old formatted tables in here.
-    let sample = var_data[0];
-    if ( Array.isArray( sample )) {
-      let rows = raw_var_data.raw();
-      var_data = rows.map(( row ) => {
-        return { var: row[0], value: row[1], checked: row[2] };
-      });
-    }
+    * Table must have headers. Arrays are not supported. */
```

Comment:
I made an issue for properly implementing supporting arrays, but as no one's using them right now, I don't feel it's urgent.